customisation/alignment.md

```diff
@@ -1,6 +1,40 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Alignment
-description:
+description: Linking corresponding units in the source and target languages
 ---
+
+**Alignment** is the process of identifying and linking the corresponding text units in the source and target languages.
+Data sets can be aligned at the word, phrase, or sentence level.
+
+Alignment can be used to create [parallel data](/customisation/parallel-data.md).
+The aligned parallel corpora are then used to train machine translation models.
+The goal is to help the machine translation system accurately translate text from one language to another by recognising patterns and regularities in the data. 
+
+#### Example
+
+Sentences are [split](/concepts/sentence-splitting.md) into smaller [tokens](/concepts/token.md).
+
+English: `The` `book` `is` `on` `the` `table` `.`
+
+German: `Das` `Buch` `liegt` `auf` `dem` `Tisch` `.`
+
+By identifying the corresponding words, such as `book` and `Buch` or `table` and `Tisch`, the two example sentences are aligned and used as [training data](/customisation/training-data.md) for the machine translation system.
+
+### Approaches
+
+Machine translation systems use various alignment approaches to link two data sets at different granularity levels.
+
+- In manual alignment, bilingual human translators align corresponding text [segments](/concepts/segment.md) in the source and target languages.
+- [Rule-based machine translation](/approaches/rule-based-machine-translation.md) uses linguistic rules and patterns to align words and phrases in two languages.
```

Comment:
To avoid repetition:

- [Rule-based machine translation](/approaches/rule-based-machine-translation.md) uses linguistic rules and patterns.